nucliadb/requirements.txt

```diff
@@ -64,4 +63,4 @@ async_lru>=2.0.4
 async-timeout>=4.0.3
 cachetools>=5.3.2
 types-cachetools>=5.3.0.5
-kubernetes_asyncio
+kubernetes_asyncio<30.0.0
```

Comment:
Why this constraint?

Reply:
because when I ran `pdm update` it bumped to 30.x and some tests stopped working